src/components/Playlist/Playlist.js

```diff
@@ -0,0 +1,78 @@
+import { useVideo } from "./../../contexts/video-context";
+import "./playlist.css";
+import { v4 as uuidv4 } from "uuid";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { NavLink } from "react-router-dom";
+
+export function Playlist() {
+  const { state, dispatch, youtubePlayer, setYoutubePlayer } = useVideo();
+  const playlistItem = Object.keys(state.playList);
+
+  return (
+    <div className="playlist">
+      {playlistItem.map((item) => (
+        <li className="playlist-item" key={uuidv4()}>
+          <div className="playlist-title">
+            <h1>{item}</h1>
+            <FontAwesomeIcon
+              icon={faTrash}
+              id="delete-icon"
+              title="delete playlist"
+              onClick={() => dispatch({ type: "DELETE_PLAYLIST", item })}
+            />
+          </div>
+          {state.playList[item].length > 0 ? (
+            <div className="playlist-videos">
+              {state.playList[item].map((video) => (
+                <div className="video-item playlist-video" key={video.id}>
+                  <img
+                    src={video["snippet"]["thumbnails"].standard.url}
+                    alt={video.snippet.title}
+                  />
+                  <NavLink
+                    to={`/video?id=${video.id}`}
+                    style={{ textDecoration: "none", color: "white" }}
+                  >
+                    <div className="video-desc playlist-play">
```

Comment:
why are you using NavLink here? shouldn't we use Link?

Reply:
ya i guess it is the same thing!